app/constants/repos/repo-schema.ts

```diff
@@ -0,0 +1,8 @@
+// run `yarn jsonschema` after making changes to the file
+// which updates repo-schema.json
+
+import { InternalSidebarConfig } from "~/ui/design-system/src/lib/Components/InternalSidebar"
+
+export type RepoSchema = {
+  sidebar: InternalSidebarConfig
+}
```

Comment:
initially i went with a schema that was similar to docusaurus', but realized it would make more sense to use our existing types in the InternalSidebar component